Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RazorTargetTests.cs

```diff
@@ -40,7 +40,7 @@ public class RazorTargetTests
         public TestContext TestContext { get; set; }
 
         [TestMethod]
-        public void Razor_CheckErrorLogProperties()
```

Comment:
the diff isn't very smart :
-  I added a new test `Razor_WhenNoSonarErrorLog_NoPropertiesAreSet`
- I renamed `Razor_CheckErrorLogProperties` to `Razor_WhenSonarErrorLogSet_SetsRazorErrorLogProperties`
- `SonarPrepareRazorProjectCodeAnalysis_WithSourceGenerators_NotExecuted` becomes a DataTestMethod test